InstallingDrivers.md

```diff
@@ -120,3 +120,78 @@ _Note: Only Skylake, Kaby Lake, and Coffee Lake offer full Vulkan support. Broad
 To install support for Vulkan API (will be functional only if you have a [Vulkan capable GPU](https://en.wikipedia.org/wiki/Vulkan_(API)#Compatibility) and driver), execute following command:
 
     sudo dnf install vulkan-loader vulkan-loader.i686
+
+    
+## openSUSE Tumbleweed/Leap
+
+First of all enabling multiarch or 32bit support is not required here it should not be disbaled in the first place.
```

Comment:
How so? There are a lot of 32-bit apps out there that need 32-bit driver support (both for GL and Vulkan), plus wine itself depends on many 32-bit libraries to run them in the first place.

Reply:
As far as I understand multiarch on Ubuntu and such means that they enable the installation of 32bit libraries. On openSUSE they are available by default and also I included 32bit packages below for ensuring 32bit dependencies.

I only added that line since it may cause confusion that enabling multi arch is required on other distros but no instructions are given for openSUSE because that's not necessary here since 32bit packages are not restricted in Leap nor Tumbleweed.

Reply:
Oh, well then it shouldn't really be emphasized honestly.